extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ConfigBuildStep.java

```diff
@@ -185,47 +195,43 @@ void validateConfigInjectionPoints(ValidationPhaseBuildItem validationPhase,
                     propertyDefaultValue = defaultValue.asString();
                 }
 
-                configProperties.produce(new ConfigPropertyBuildItem(propertyName, injectedType, propertyDefaultValue));
+                // Observers requires validation at static init
```

Comment:
This is IMO not correct. Observes in general do not require validation during `STATIC_INIT`. Are you talking specifically about `@Observes @Initialized(ApplicationScoped.class)`?

Reply:
Yes.

Reply:
In that case, the comment should be updated accordingly...

Reply:
Sure.